src/main/java/com/ninecookies/wiremock/extensions/util/Objects.java

```diff
@@ -2,11 +2,17 @@
 
 import java.util.regex.Matcher;
 
+import com.github.tomakehurst.wiremock.common.Json;
 import com.github.tomakehurst.wiremock.http.Response;
 import com.jayway.jsonpath.DocumentContext;
 
 public class Objects {
 
+    public static <T> T convert(Object source, Class<T> destinationType) {
```

Comment:
Taste: I would drop a javadoc saying that the target type has to have the same structure/fields than source and that it basically copies attributes. Does it fail if attributes are missing?